acceptance/tests/pxp-module-puppet/run_puppet_agent_non_root.rb

```diff
@@ -8,22 +8,29 @@
     skip_test "Test is not compatible with #{platform}" if platform =~ /windows/
 
     step 'create non-root user on all nodes' do
-      @user_home_dir = '/foo'
+      @user_name = 'foo'
+      @group_name = 'foobar'
+      (platform =~ /solaris/) ? @user_home_dir = "/export/home#{@user_name}" : @user_home_dir = "/home/#{@user_name}"
```

Comment:
How about `@user_home_dir = platform =~ /solaris/ ? "/export/home#{@user_name}" : "/home/#{@user_name}"`?

Reply:
Thats much cleaner, I guess that's what I get for letting my IDE refactor my code.  :)